alpha/apps/kaltura/lib/kContextDataHelper.php

```diff
@@ -210,23 +205,22 @@ private function setContextDataFlavorAssets($flavorTags)
 
 		$flavorParamsIds = null;
 		$flavorParamsNotIn = false;
-		if(!$this->isAdmin)
-		{
-			foreach ($this->contextDataResult->getActions() as $action) 
-			{	
-				if($action->getType() == RuleActionType::BLOCK)
-				{
-					//in case of block action do not set the list of flavors
-					return;
-				}
-				if($action->getType() == RuleActionType::LIMIT_FLAVORS)
-				{
-					/* @var $action kAccessControlLimitFlavorsAction */
-					$flavorParamsIds = explode(',', $action->getFlavorParamsIds());
-					$flavorParamsNotIn = $action->getIsBlockedList();
-				}
-			}	
-		}
+		
+		foreach ($this->contextDataResult->getActions() as $action) 
```

Comment:
This is ok that in case of optimized_playback it wont run. if I remember correctly this was the entire point of this setup to avoid running ACL for calls made by the player but do run them for testme|clietnLibs|etc ...

Previously this was also the behavior if we were running with user KS and we had access control we would also have skipped the applyContext.